src/languages/en.ts

```diff
@@ -808,6 +808,14 @@ export default {
             setDefaultFailure: 'Something went wrong. Please chat with Concierge for further assistance.',
         },
         addBankAccountFailure: 'An unexpected error occurred while trying to add your bank account. Please try again.',
+        cardDetails: {
```

Comment:
What do you think about moving this block under `cardPage`?

Reply:
Great idea, on it!